CoursesManager.Tests/CoursesManager.Tests.csproj

```diff
@@ -32,4 +32,8 @@
 		<Using Include="NUnit.Framework" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <Folder Include="Course\" />
+	</ItemGroup>
+
```

Comment:
Waar is dit voor?

Reply:
Voor unit tests van cursus (deed ik in verkeerde branch lol)